lighthouse/environment/src/lib.rs

```diff
@@ -358,14 +357,13 @@ impl<E: EthSpec> Environment<E> {
         }
     }
 
-    /// Returns a `Context` where the `service_name` is added to the logger output.
-    pub fn service_context(&self, service_name: String) -> RuntimeContext<E> {
+    /// Returns a context.
+    pub fn service_context(&self) -> RuntimeContext<E> {
```

Comment:
This method currently has the same implementation as `core_context` and does not appear to be used anywhere. It can likely be removed.